cmd/ci-secret-bootstrap/config.yaml

```diff
@@ -0,0 +1,96 @@
+- from:
+    ci-operator:
+      bw_item: sentry
+      field: ci-operator
+  to:
+    - cluster: default
+      namespace: ci
+      name: sentry-dsn
+    - cluster: build01
+      namespace: ci
+      name: sentry-dsn
+- from:
+    service-account.json:
+      bw_item: gcs-publisher
+      attachment: credentials.json
+  to:
+    - cluster: default
+      namespace: ci
+      name: gce-sa-credentials-gcs-publisher
+    - cluster: build01
+      namespace: ci
+      name: gce-sa-credentials-gcs-publisher
+- from:
+    sa.ci-operator.apici.config:
+      bw_item: build_farm
+      attachment: sa.ci-operator.apici.config
+  to:
+    - cluster: default
+      namespace: ci
+      name: apici-ci-operator-credentials
+    - cluster: build01
+      namespace: ci
+      name: apici-ci-operator-credentials
+- from:
+    pull-secret:
+      bw_item: quay.io
+      attachment: pull-secret
+    insights-live.yaml:
+      bw_item: insights-ci-account
+      attachment: insights-live.yaml
+    .awscred:
+      bw_item: jenkins-ci-iam
+      attachment: .awscred
+    ssh-privatekey:
+      bw_item: jenkins-ci-iam
+      attachment: ssh-privatekey
+    ssh-publickey:
+      bw_item: jenkins-ci-iam
+      attachment: ssh-publickey
+    ops-mirror.pem:
+      bw_item: mirror.openshift.com
+      attachment: cert-key.pem
+  to:
+    - cluster: default
+      namespace: ci
+      name: cluster-secrets-aws
+    - cluster: build01
+      namespace: ci
+      name: cluster-secrets-aws
+- from:
+    ci.htpasswd:
+      bw_item: boskos-oauth-proxy
+      attachment: ci.htpasswd
+    password:
+      bw_item: boskos-oauth-proxy
+      field: boskos-password
+    username:
+      bw_item: boskos-oauth-proxy
+      field: boskos-username
+  to:
+    - cluster: default
+      namespace: ci
+      name: boskos-credentials
+    - cluster: build01
+      namespace: ci
+      name: boskos-credentials
+- from:
+    ops-mirror.pem:
+      bw_item: mirror.openshift.com
+      attachment: cert-key.pem
+    rh-cdn.pem:
+      bw_item: rh-cdn
+      attachment: rh-cdn.pem
+  to:
+    - cluster: default
+      namespace: ci
+      name: mirror.openshift.com
+    - cluster: default
+      namespace: ocp
+      name: mirror.openshift.com
+    - cluster: build01
+      namespace: ocp
+      name: mirror.openshift.com
```

Comment:
@stevekuznetsov 

Should I throw an error on this case?
Same content from BW is mapped to 2 secrets in the same cluster?
The sync of secrets inside a cluster can be done with ci-secret-mirroring-controller.

Reply:
The mirroring controller was built in order to allow a user who does not have access to production namespaces to  create a secret somewhere else and have it mirrored in, allowing them to manage it. That use-case is distinct from happening to have two secrets on the cluster that are from the same data. This should not error and we should be OK to populate from BitWarden.

Reply:
It does the some overlapping jobs with

https://github.com/openshift/release/blob/master/core-services/secret-mirroring/_mapping.yaml#L171-L176